packages/main/src/themes/Token.css

```diff
@@ -41,7 +41,7 @@
 :host([selected]:not([readonly]):hover) {
 	background: var(--_ui5_token_selected_hover_background);
 	border-color: var(--_ui5_token_selected_hover_border_color);
-	box-shadow: var(--sapContent_Interaction_Shadow);
+	box-shadow: var(--_ui5_token_selected_box_shadow);
```

Comment:
Do we need a variable? I noticed that for all the Horizon themes its value is sapContent_Selected_Shadow and in the base file it is sapContent_Interaction_Shadow - but is sapContent_Interaction_Shadow correct and for which theme it is specified?